src/components/Input/index.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import Input from '.';
+
+const InputTestData = {
+  label: 'Sign in',
+  dataTestId: 'test-input',
+};
+
+describe('Input', () => {
+  it('render the label correctly', () => {
+    render(<Input label={InputTestData.label} type="text" name="email" data-test-id={InputTestData.dataTestId}></Input>);
+    const label = screen.getByText(InputTestData.label);
+    expect(label).toBeVisible();
+  });
```

Comment:
Can we combine the two tests together? I don't think the component's complexity warrants having really granular tests IMO 

And also adding a bit of spacing between the lines goes a long way for readability 😄 

```suggestion
  it('renders the component', () => {
    render(<Input label={InputTestData.label} type="text" name="email" data-test-id={InputTestData.dataTestId}></Input>);

    const label = screen.getByText(InputTestData.label);
    const input = screen.getByRole('textbox');

    expect(label).toBeVisible();
    expect(input).toBeVisible();
    
    expect(input).toHaveAttribute('class', 'form-field__input');
  });
```